src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -5135,6 +5265,84 @@ public void PartialMember_DeleteInsert_ConstructorWithInitializers()
                 });
         }
 
+        [Fact]
+        public void PartialMember_DeleteInsert_MethodAddParameter()
+        {
+            var srcA1 = "partial struct S { }";
+            var srcB1 = "partial struct S { void F() {} }";
+            var srcA2 = "partial struct S { void F(int x) {} }";
+            var srcB2 = "partial struct S { }";
+
+            EditAndContinueValidation.VerifySemantics(
+                new[] { GetTopEdits(srcA1, srcA2), GetTopEdits(srcB1, srcB2) },
+                new[]
+                {
+                    DocumentResults(
+                        semanticEdits: new[]
+                        {
+                            SemanticEdit(SemanticEditKind.Insert, c => c.GetMember("S.F"))
+                        }),
+
+                    DocumentResults(
+                        diagnostics: new[]
+                        {
+                            Diagnostic(RudeEditKind.Delete, "partial struct S", DeletedSymbolDisplay(FeaturesResources.method, "F()"))
+                        })
+                });
+        }
+
+        [Fact]
+        public void PartialMember_DeleteInsert_UpdateMethodParameterType()
+        {
+            var srcA1 = "partial struct S { }";
+            var srcB1 = "partial struct S { void F(); }";
```

Comment:
This should have an `int x` parameter I'm guessing, otherwise the name of the test is wrong, and its the same as the above test anyway

Reply:
You are right, `int x` is missing.